Utils.py

```diff
@@ -35,7 +35,7 @@
 
 
 def tuplize_version(version: str) -> Version:
-    return Version(*(int(piece, 10) for piece in version.split(".")))
```

Comment:
:+1: I originally thought that the `, 10` was to make it more strict, but after double-checking the docs, you can not hit the other int() variations without violating the typing.
(If piece is str, you'll always hit the int(str, base=10) overload).